web/modules/custom/dpl_react_apps/dpl_react_apps.module

```diff
@@ -189,6 +189,7 @@ function dpl_react_apps_texts(): array {
     'patron-page-change-pincode-body' => t('Change current pin by entering a new pin and saving', [], ['context' => 'Global']),
     'patron-page-change-pincode-header' => t('PINCODE', [], ['context' => 'Global']),
     'patron-page-confirm-pincode-label' => t('Confirm new pin', [], ['context' => 'Global']),
+    'patron-page-phone-input-message' => t('The phone number must be 6 to 15 characters in length and should be comprised solely of numbers or begin with a +', [], ['context' => 'Global']),
```

Comment:
Is this a deliberate change?

Reply:
Yes. I moved the string to where the other registration form strings are stored. 
Before this, we got an error as react couldn't find it.